float64 latitude
float64 longitude
float64 azimuth
float64 roll
float64 pitch
